src/Collection/Collection.model.ts

```diff
@@ -69,16 +70,34 @@ export class Collection extends Model<CollectionAttributes> {
     q,
     assignee,
     status,
+    type,
     address,
     thirdPartyIds,
     remoteIds,
   }: Pick<
     FindCollectionParams,
-    'q' | 'assignee' | 'status' | 'address' | 'thirdPartyIds' | 'remoteIds'
+    | 'q'
+    | 'assignee'
+    | 'status'
+    | 'type'
+    | 'address'
+    | 'thirdPartyIds'
+    | 'remoteIds'
   >) {
-    if (!q && !assignee && !status && !address && !thirdPartyIds?.length) {
+    if (
+      !q &&
+      !assignee &&
+      !status &&
+      !type &&
+      !address &&
+      !thirdPartyIds?.length
+    ) {
       return SQL``
     }
+    const isStandard =
+      'collections.third_party_id is NULL AND collections.urn_suffix is NULL'
```

Comment:
```suggestion
      SQL`collections.third_party_id is NULL AND collections.urn_suffix is NULL`
```
Let's wrap it up with the `SQL` method so we don't have to use the `raw` down below :)